float64[16] pose
---
sensor_msgs/Image label
bool success
string error_msg